Modelica_LinearSystems2/Math/Matrices/Internal/QR.mo

```diff
@@ -18,7 +18,7 @@ algorithm
      + "] has more columns as rows.
 This is not allowed when calling Modelica.Matrices.QR(A).");

```

Comment:
```suggestion
This is not allowed when calling Modelica.Math.Matrices.QR(A).");

```

Was also wrong in copied function Modelica.Math.Matrices.QR and fixed there by https://github.com/modelica/ModelicaStandardLibrary/commit/412d17023bd7613ef226a611271f5001cbb54e78.